tests/test_entities.py

```diff
@@ -2204,6 +2230,53 @@ def test_creation_and_update(self):
         self.assertEqual(expected_dct, cfg_template.update_payload())
 
 
+class TemplateInputTestCase(TestCase):
+    """Tests for :class:`nailgun.entities.JobTemplate`."""
+    def setUp(self):
+        self.entity = entities.TemplateInput(config.ServerConfig('some url'), template='1')  # noqa  pylint: disable=line-too-long
+        self.read_patcher = mock.patch.object(EntityReadMixin, 'read')
+        self.read = self.read_patcher.start()
+        self.read.return_value = entities.TemplateInput(
+            config.ServerConfig('some url'),
+            template=1,
+            name='my new template input',
+            input_type='user')
+        self.read_json_patcher = mock.patch.object(
+            EntityReadMixin,
+            'read_json')
+        self.read_json = self.read_json_patcher.start()
+
+    def tearDown(self):
+        self.read_json_patcher.stop()
+        self.read_patcher.stop()
+
+    def test_read(self):
+        entities.TemplateInput(config.ServerConfig('some url'), id='1',name='my new template input', template='1').read(attrs={'name':'my new template input',  'template': '1','input_type':'user'}) # noqa  pylint: disable=line-too-long
```

Comment:
```python
entities.TemplateInput(
            config.ServerConfig('some url'),
            id='1',
            name='my new template input',
            template='1').read(
            attrs={
                'name': 'my new template input',
                'template': '1',
                'input_type': 'user'
            }
        )
```
same for other long lines and remove the "# noqa  pylint: disable=line-too-long" from any line

Reply:
no more `"# noqa pylint: disable=line-too-long` in `test_entities.py`.